### PR TITLE
Filling out Papers Page

### PR DESCRIPTION
**Pull request by KevinCawley**
Copied the Papers from the docs to the website. 
**Description**

Copied the papers from https://tardis-sn.github.io/tardis/research/research_done_using_TARDIS/research_papers.html 
to the papers page of the website.
**Motivation and context**

**How has this been tested?**
- [x] Testing pipeline.
- [ ] Other. 

**Examples**

**Type of change**

- [ ] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

**Comment by KevinCawley** [comment]
https://kevincawley.github.io/tardis-sn.github.io/branch/papers/research/papers/

File: content/research/papers.md [hunk 1]
```diff
@@ -4,4 +4,147 @@ date: 2021-05-28T09:27:27-05:00
 draft: false
 layout: page
 ---
-This page is under construction
\ No newline at end of file
+## Papers Using Tardis
```
Comment (jaladh-singhal) [inline content/research/papers.md:7]:
```suggestion
## Papers using *tardis* package
```

File: content/research/papers.md [hunk 2]
```diff
@@ -4,4 +4,147 @@ date: 2021-05-28T09:27:27-05:00
...
+**Young, P. R., Dere, K. P., Landi, E., et al.** 2016, JPhB, *"The CHIANTI atomic database"* <a href="https://ui.adsabs.harvard.edu/abs/2016JPhB...49g4009Y" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Noebauer, U. M., Sim, S. A.** 2015, MNRAS, *"Self-consistent modelling of line-driven hot-star winds with Monte Carlo radiation hydrodynamics"* <a href="https://ui.adsabs.harvard.edu/abs/2015MNRAS.453.3120N" target="_blank" rel="noopener nofollow">(ADS Link) </a>
+
+
+**Matthews, J. H., Knigge, C., Long, K. S., et al.** 2015, MNRAS, *"The impact of accretion disc winds on the optical spectra of cataclysmic variables"* <a href="https://ui.adsabs.harvard.edu/abs/2015MNRAS.450.3331M" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Kerzendorf, Wolfgang E., Sim, Stuart A.** 2014, MNRAS, *"A spectral synthesis code for rapid modelling of supernovae"* <a href="https://ui.adsabs.harvard.edu/abs/2014MNRAS.440..387K" target="_blank" rel="noopener nofollow">(ADS Link)</a>
```
Comment (jaladh-singhal) [inline content/research/papers.md:150]: Add section for carsus package and fill in with item no. 1 from here: https://tardis-sn.github.io/carsus/research/publications.html (no. 2 is actually a poster so we don't want it here)

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]
Looks neat - good to see that you've used markdown. There are few minor things

File: content/research/papers.md [hunk 3]
```diff
@@ -4,4 +4,152 @@ date: 2021-05-28T09:27:27-05:00
 draft: false
 layout: page
 ---
-This page is under construction
\ No newline at end of file
+## Papers using the *TARDIS* package
```
Comment (jaladh-singhal) [inline content/research/papers.md:7]:
```suggestion
## Papers using the *tardis* package
```

File: content/research/papers.md [hunk 4]
```diff
@@ -4,4 +4,152 @@ date: 2021-05-28T09:27:27-05:00
...
+**Noebauer, U. M., Sim, S. A.** 2015, MNRAS, *"Self-consistent modelling of line-driven hot-star winds with Monte Carlo radiation hydrodynamics"* <a href="https://ui.adsabs.harvard.edu/abs/2015MNRAS.453.3120N" target="_blank" rel="noopener nofollow">(ADS Link) </a>
+
+
+**Matthews, J. H., Knigge, C., Long, K. S., et al.** 2015, MNRAS, *"The impact of accretion disc winds on the optical spectra of cataclysmic variables"* <a href="https://ui.adsabs.harvard.edu/abs/2015MNRAS.450.3331M" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Kerzendorf, Wolfgang E., Sim, Stuart A.** 2014, MNRAS, *"A spectral synthesis code for rapid modelling of supernovae"* <a href="https://ui.adsabs.harvard.edu/abs/2014MNRAS.440..387K" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+## Papers using the *CARSUS* package
```
Comment (jaladh-singhal) [inline content/research/papers.md:153]:
```suggestion
## Papers using the *carsus* package
```

**Review by jaladh-singhal — APPROVED** [review]

**Pull request merged**